tpot/base.py

```diff
@@ -263,6 +265,7 @@ def __init__(self, generations=100, population_size=100, offspring_size=None,
         self.verbosity = verbosity
         self.disable_update_check = disable_update_check
         self.random_state = random_state
+        self._file = progress_file
```

Comment:
Based on scikit-learn API, please also use `self.log_file = log_file` in __init__.